docs/CLI.md

```diff
@@ -275,7 +281,7 @@ Lists all test files that Jest will run given the arguments, and exits.
 
 ### `--logHeapUsage`
 
-Logs the heap usage after every test. Useful to debug memory leaks. Use together with `--runInBand` and `--expose-gc` in node.
+Logs the heap usage after every test. Useful to debug memory leaks. Can be used together with `--runInBand` and `--expose-gc` in node to diagnose memory leaks, etc.
```

Comment:
Perhaps 'or'? Reading comment from the user it sounded like 'and' was the misleading part. At least this is how my eye is reading this 'and'.

Also would be good to make this change in version docs as well. They live in [website/versioned_docs](https://github.com/facebook/jest/tree/main/website/versioned_docs)


```suggestion
Logs the heap usage after every test. Useful to debug memory leaks. Can be used together with flags like `--runInBand` or `--expose-gc` to diagnose memory leaks, etc.
```